packages/reporter/src/commands/command-model.ts

```diff
@@ -100,15 +117,16 @@ export default class Command extends Instrument {
     this.numElements = props.numElements
     this.renderProps = props.renderProps || {}
     this.timeout = props.timeout
-    this.visible = props.visible
+    // command log that are not associated with elements will not have a visibility
+    // attribute set. i.e. cy.visit(), cy.readFile() or cy.log()
+    this.isInvisible = props.visible !== undefined && !props.visible
```

Comment:
While this gets rid of the redundancy, how about sticking with `model.visible` since it mirrors the props? So here and in `update`, it could be `this.visible = props.visible === true || props.visible === undefined` instead.

Reply:
I originally changed it to reflect the boolean value and to reflect it's usage in the UI, but I change it back: [`6f5a7d9` (#20857)](https://github.com/cypress-io/cypress/pull/20857/commits/6f5a7d94ce7f26c0ac392957bd2797ad5facbe89)